src/detect-pcre.c

```diff
@@ -822,6 +836,25 @@ static int DetectPcreParseCapture(const char *regexstr, DetectEngineCtx *de_ctx,
     return -1;
 }
 
+#ifdef PCRE_HAVE_JIT_EXEC
+static void *DetectPcreThreadInit(void *data /*@unused@*/)
+{
+    pcre_jit_stack *jit_stack = pcre_jit_stack_alloc(PCRE_JIT_MIN_STACK, PCRE_JIT_MAX_STACK);
+
+    if (jit_stack == NULL) {
+        SCLogError(SC_ERR_MEM_ALLOC, "unable to allocate PCRE jit stack");
```

Comment:
does the pcre API provide a more detailed error output?

Reply:
The `pcre2` API has an interface but not `pcre`.